x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/script/ScriptTemplate.java

```diff
@@ -63,12 +81,27 @@ public Script toPainless() {
 
     private String bindTemplate() {
         List<String> binding = params.asCodeNames();
-        return binding.isEmpty() ? template : format(Locale.ROOT, template, binding.toArray());
+
+        String header = null;
+        // sort functions alphabetically
```

Comment:
I *think* I know why you want them sorted alphabetically but I think it'd be nice to have it in a comment.